include/aspect/assembly.h

```diff
@@ -377,7 +377,7 @@ namespace aspect
          * - The copy object into which assemblers add up their contributions.
          */
         boost::signals2::signal<void (const double,
-                                      internal::Assembly::Scratch::StokesPreconditioner<dim>  &,
+                                      internal::Assembly::Scratch::StokesPreconditioner<dim> &,
```

Comment:
Which version of astyle are you using.  I have 2.03, which indent refuses to run with.  It says 2.04 is required, but here are now:

astyle 2.05.1	2014-12-11	

astyle 2.05	2014-11-17

astyle 2.04	2013-11-01